test/webserv/integration/test_connection_use_socket.py

```
@@ -2,7 +2,7 @@
 import time
 import unittest
 from http.client import HTTPConnection, HTTPException
-from typing import Optional
+from typing import List, Optional
```

Ruff

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)